index.html

```diff
@@ -3379,7 +3837,16 @@ <h2>
       <p>
         This specification relies on several other underlying specifications.
       </p>
-      <dl>
+      <dl data-sort="">
+        <dt>
+          Infra
+        </dt>
+        <dd>
+          The [[!INFRA] specification defines how to <dfn data-cite=
+          "!INFRA#strip-leading-and-trailing-ascii-whitespace">strip leading
+          and trailing ascii whitespace</dfn>. It also defines the concpet of a
+          <dfn data-cite="!INFRA#list">list</dfn>.
```

Comment:
If the input form allows a wider range of UTF-8 encoded Unicode characters (not just the ASCII range), then stripping only ASCII whitespace will not necessarily yield the hoped-for result here because a fair number of Unicode code points have a Unicode general category of "Zs". For example, U+1680 (OGHAM SPACE MARK) would not get stripped unless non-ASCII space gets converted to ASCII space.

Reply:
yeah, that's a good point... specially because this might be coming from free form input from a end-user (e.g., "region"). 

@domenic, seems JS `String.prototpye.trim()` does mostly the right thing (some bugginess across browsers). Do you have any suggestions?

Reply:
Here is a test string:
```
var whiteSpace = 
"\u0009\u000A\u000B\u000C\u000D\u0020\u0085\u00A0\u1680\u2000\u2001\u2002\u2003\u2004\u2005\u2006\u2007\u2008\u2009\u200A\u2028\u2029\u202F\u205F\u3000"
whiteSpace.trim().length; // 1, because u0085 is apparently not WS. 
```

Reply:
We never trim Unicode whitespace on the platform. Only ASCII whitespace is considered as "allowed sloppiness" in argument values.

An alternative is to not remove such whitespace at all.

Reply:
@domenic Hmm. Do we assume that non-ASCII Unicode code points with a General Category of "Zs" will be filtered out somehow? (For example, by conversion of such code points to SPACE [U+0020]?) If this is guidance that the i18n WG has provided in the past, perhaps I've just missed the pointer. I can ask in the next i18n WG call if needed.

Reply:
Generally if a string contains Unicode whitespace, we don't try to second-guess the user or developer, and just pass it through.

Reply:
Thanks for the clarification! I just asked about it on the www-international list, too: https://lists.w3.org/Archives/Public/www-international/2018JanMar/0042.html

Reply:
I'm ok with doing the minimal ASCII whitespace stripping, for platform consistency. Developers can then call `.trim()` to strip out the rest if so desired.